nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/autodiff/samediff/ops/SDBaseOps.java

```diff
@@ -2867,6 +2867,32 @@ public SDVariable reverseSequence(String name, SDVariable x, SDVariable seq_leng
     return sd.updateVariableNameAndReference(out, name);
   }
 
+  /**
+   * Element-wise scalar floor division operation: out = floorDiv(in, value).<br>
+   *
+   * @param in Input variable (NUMERIC type)
+   * @param value Scalar value to compare
+   * @return output Output variable (NUMERIC type)
+   */
+  public SDVariable scalarFloorDiv(SDVariable in, double value) {
+    SDValidation.validateNumerical("scalarFloorDiv", "in", in);
+    return new org.nd4j.linalg.api.ops.impl.scalar.ScalarDivision(sd,in, value).outputVariable();
+  }
+
+  /**
+   * Element-wise scalar floor division operation: out = floorDiv(in, value).<br>
+   *
+   * @param name name May be null. Name for the output variable
+   * @param in Input variable (NUMERIC type)
+   * @param value Scalar value to compare
+   * @return output Output variable (NUMERIC type)
+   */
+  public SDVariable scalarFloorDiv(String name, SDVariable in, double value) {
+    SDValidation.validateNumerical("scalarFloorDiv", "in", in);
+    SDVariable out =  new org.nd4j.linalg.api.ops.impl.scalar.ScalarDivision(sd,in, value).outputVariable();
+    return sd.updateVariableNameAndReference(out, name);
+  }
```

Comment:
Let's put this in Math namespace with name `floorDiv`

I also just realized we're missing things like add, mul, etc - those should all go into Math namespace

Reply:
Moved floorDiv, floorMod.